general/1_terminal/mykola-kulinich/t4.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/zsh
+
+jq '.messages[] | .text' $1 > t4.txt 
```

Comment:
I'll accept this, but if you'd like an additional assignment, try to make it look prettier.

Currently, it outputs arrays when it's a forward, mention or a link.